packages/android_alarm_manager/ios/android_alarm_manager.podspec

```diff
@@ -4,14 +4,17 @@
 Pod::Spec.new do |s|
   s.name             = 'android_alarm_manager'
   s.version          = '0.0.1'
-  s.summary          = 'A new flutter plugin project.'
+  s.summary          = 'Flutter Android Alarm Manager'
   s.description      = <<-DESC
-A new flutter plugin project.
+A Flutter plugin for accessing the Android AlarmManager service, and running Dart code in the background when alarms fire.
+This plugin a no-op on iOS. 
+Downloaded by pub (not CocoaPods).
                        DESC
-  s.homepage         = 'http://example.com'
-  s.license          = { :file => '../LICENSE' }
-  s.author           = { 'Your Company' => 'email@example.com' }
-  s.source           = { :path => '.' }
+  s.homepage         = 'https://github.com/flutter/plugins'
+  s.license          = { :type => 'BSD', :file => '../LICENSE' }
+  s.author           = { 'Flutter Dev Team' => 'flutter-dev@googlegroups.com' }
+  s.source           = { :http => 'https://github.com/flutter/plugins/tree/master/packages/android_alarm_manager' }
```

Comment:
This is a bogus source since there's nothing to download at this URL and downloading is handled by pub, not CocoaPods tooling.